apps/src/code-studio/pd/professional_learning/RegionalWorkshopCatalogCard.jsx

```diff
@@ -77,143 +70,109 @@ const RegionalWorkshopCatalogCard = ({
       },
       PLATFORMS.BOTH
     );
-    navigateToHref(
-      customRegistrationLink
-        ? customRegistrationLink
-        : `/pd/workshops/${id}/enroll`
-    );
+    navigateToHref(`/professional-learning/workshops/${id}`);
   };
 
   return (
-    <>
```

Comment:
I recommend viewing with hidden whitespace since a lot of this is just indentation from removing the dialog and the partial `<>`